src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -191,122 +155,141 @@ public String  MakeGUID() {
 
     public void BeginConnectPeer(String toPeerId, ConnectStatusCallback connectStatusCallback) {
         ConnectStatusCallback tmpCallback = connectStatusCallback;
```

Comment:
Why is this tmpCallback variable created? No one can change the pointer to connectStatusCallback from under you since it's local. And you don't use connectStatusCallback for anything but setting tmpCallback. So I'm confused why it's here.